src/main/java/models/Administrator.java

```diff
@@ -14,10 +14,10 @@ public Administrator(
       int registerID) {
 
     super(name, birthCity, gender, username, password, email);
-    this.registerID = registerID;
+    this.registerId = registerID;
```

Comment:
Ficou com o ID maiúsculo. Se foi proposital, então não precisa usar o this em registerId, porque o java já iria considerar com duas variáveis diferentes.